index.html

```diff
@@ -173,6 +173,57 @@ <h3>Skills</h3>
           </li>
         </ul>
       </section>
+      <footer  class="contact-form-section d-flex flex-center">
+        <div class="form-sub-container">
+          <h2 class="h2-size">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+          <form action="https://formspree.io/f/xnqlovnj" method="post">
+            <div>
+              <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+            </div>
+            <div>
+              <input type="email" id="mail" name="user_email" placeholder="Email-address" required>
+            </div>
+            <div>
+              <textarea id="msg" name="user_message" placeholder="Enter text here" rows="6" maxlength="500" required></textarea>
+            </div>
+            <div>
+              <button type="submit" class="footer-btn">Get in touch</button>
```

Comment:
- [ ] It would be better if we apply the `cursor: pointer` interactions so that the user will know when to click and when not to click